bigtable-hbase-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/filters/FuzzyRowFilterAdapter.java

```diff
@@ -86,6 +88,8 @@ private static Filter createSingleRowFilter(byte[] key, byte[] mask) throws IOEx
         baos.write(ReaderExpressionHelper.ANY_BYTE_BYTES);
       }
     }
+    // match any trailing bytes
+    baos.write(ReaderExpressionHelper.ALL_BYTE_BYTES);
```

Comment:
Looks this is the main change, everything else is clean up. If so, LGTM.

Reply:
You are correct.